### PR TITLE
Redux & Thunk added.

### PR DESCRIPTION
**Pull request by xlreon**

### CODE REVIEW

File: viber-fe/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { initialState } from "../store";
+
+function reducer(state = initialState, action) {
+    const { type, payload } = action;
+    switch (type) {
+        case AppActions.LOGIN:
+            return state
+        case AppActions.SIGNUP:
+            return state
+            default:
```
Comment (J3rry320) [inline viber-fe/reducers/index.js:11]: Check Indentation

**Review by J3rry320 — APPROVED** [review]
Looks Good! Like any normal store should! Just correct the indentation where commented

File: viber-fe/reducers/index.js (hunk 1 above)
Comment (xlreon) [reply viber-fe/reducers/index.js:11]: I am afk can you do the indent and push?

Comment (xlreon) [reply viber-fe/reducers/index.js:11]: Merge it also.

**Pull request merged**